models/kendra.go

```diff
@@ -0,0 +1,65 @@
+package models
+
+import (
+	"context"
+	"log"
+
+	"github.com/aws/aws-sdk-go-v2/service/kendra"
+)
+
+var opts = kendra.Options{
+	Credentials: prov,
+	Region:      region,
+}
+
+var client = kendra.New(opts)
+
+type KendraResult struct {
+	ImgPath string
+	Title   string
+	Link    string
+	Excerpt string
+}
+
+type KendraResults struct {
+	QueryOutput kendra.QueryOutput
+	Results     []KendraResult
+	Query       string
+	Count       int
+}
+
+func queryOutputToResults(out kendra.QueryOutput) KendraResults {
+	results := KendraResults{
+		Results: []KendraResult{},
+	}
+
+	for _, item := range out.ResultItems {
+		res := KendraResult{
+			ImgPath: "images/placeholder.jpg",
```

Comment:
should we leave a placeholder image in? I don't think we're going to get any images to replace the placeholder anytime soon

Reply:
it might be better to just not have a placeholder image

Reply:
Do you think it would be better to drop the property from the struct along with removing the image from the html or just remove the image from the html?

Reply:
Hmm, i think you can drop it from the struct and the html, since it seems like very little work to add it back